transform/source.go

```diff
@@ -10,7 +10,7 @@ type sourceType string
 const (
 	venues         sourceType = "ESTABELE"
 	motives                   = "MOTICSV"
-	main                      = "EMPRECSV"
+	base_cpnj                 = "EMPRECSV"
```

Comment:
No Go usamos _camel case_ ; )

```suggestion
	baseCNPJ                 = "EMPRECSV"
```